commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/sftp/SftpConstants.java

```diff
@@ -0,0 +1,37 @@
+package org.apache.commons.vfs2.provider.sftp;
```

Comment:
License header should be before the package name?

Reply:
Done